llpc/test/shaderdb/ObjUniformBlock_TestIndirectIndex_lit.frag

```diff
@@ -25,10 +25,10 @@ void main()
 ; SHADERTEST-LABEL: {{^// LLPC}} pipeline patching results
 ; SHADERTEST: call <4 x i32> @llvm.amdgcn.s.buffer.load.v4i32(<4 x i32> %{{[0-9]*}}, i32 0, i32 0)
 ; SHADERTEST: call i32 @llvm.amdgcn.s.buffer.load.i32(<4 x i32> %{{[0-9]*}}, i32 0, i32 0)
-; SHADERTEST: call <4 x i32> @llvm.amdgcn.s.buffer.load.v4i32(<4 x i32> %{{[0-9]*}}, i32 %{{[0-9]*}}, i32 0)
+; SHADERTEST: call <4 x i32> @llvm.amdgcn.s.buffer.load.v4i32(<4 x i32> %{{[0-9]*}}, i32 %{{[0-9a-z.]*}}, i32 0)
```

Comment:
Can you use `%{{\w+}}` everywhere instead? (I'm not actually sure if `\w` is supported but I think it should be.)

Reply:
I'll see if we cannot programmatically replace %{{[0-9]*}} in all the lit tests as a follow up PR.